### ISSUE TITLE
Annotations

### ISSUE THREAD

**Issue by daneflats** [Hyphen-ated/DJBot#44]
is it possible for the player to auto hide annotations on each new video being played?, great program by the way, loving it.

**Comment by Hyphen-ated** [comment]
Yeah that looks possible. (reference for later: "iv_load_policy" documented at https://developers.google.com/youtube/player_parameters )
In the meantime, you could disable annotations by default in your youtube settings if you want to.

**Comment by daneflats** [comment]
Thank you very much for the response, I did try setting default to off for the annotations, it works in the browser but not in the djbot viewer in browser, maybe I am doing something wrong. In any case, thank you once again.
